spec/features/team/invite_a_team_member_spec.rb

```diff
@@ -4,102 +4,134 @@
+            expect(invited_user.confirmed?).to eq(false)
+            expect(invited_user.organisation).to eq(user.organisation)
+          end
+        end
 
-      before do
-        fill_in "Email", with: invited_user_email
-      end
+        context 'with non gov.uk email address' do
+          let(:invited_user_email) { 'incorrect@gmail.com' }
+          let(:invited_user) { User.find_by(email: invited_user_email) }
+
+          it 'tells user that email must be a valid gov.uk email' do
+            expect {
+              click_on 'Send invitation email'
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(0)
+            expect(invited_user).to eq(nil)
+            expect(page).to have_content('Email must be from a government domain')
+          end
+        end
+
+        context 'with an email that already exists' do
+          let(:invited_user_email) { user.email }
 
-      context "with gov.uk email address" do
-        let(:invited_user_email) { "correct@gov.uk" }
-        let(:invited_user) { User.find_by(email: invited_user_email) }
-
-        it "creates an unconfirmed user" do
-          expect {
-            click_on "Send invitation email"
-          }.to change { User.count }.by(1)
-          expect(InviteUseCaseSpy.invite_count).to eq(1)
-          expect(invited_user.confirmed?).to eq(false)
-          expect(invited_user.organisation).to eq(user.organisation)
+          it 'tells the user that the email has already been taken' do
+            expect {
+              click_on 'Send invitation email'
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(0)
+            expect(page).to have_content('Email has already been taken')
+          end
         end
-      end
 
-      context "with non gov.uk email address" do
-        let(:invited_user_email) { "incorrect@gmail.com" }
-        let(:invited_user) { User.find_by(email: invited_user_email) }
-
-        it "tells user that email must be a valid gov.uk email" do
-          expect {
-            click_on "Send invitation email"
-          }.to change { User.count }.by(0)
-          expect(InviteUseCaseSpy.invite_count).to eq(0)
-          expect(invited_user).to eq(nil)
-          expect(page).to have_content("Email must be from a government domain")
+        context 'without an email' do
+          let(:invited_user_email) { '' }
+
+          it 'tells the user that email cannot be blank' do
+            expect {
+              click_on 'Send invitation email'
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(0)
+            expect(page).to have_content("Email can't be blank")
+          end
         end
-      end
 
-      context "with an email that already exists" do
-        let(:invited_user_email) { user.email }
+        context 'tells user that email must be a valid gov.uk email' do
```

Comment:
This looks like it already exists on Line 47: `context 'with non gov.uk email address' do`